rcmgr.go

```diff
@@ -15,6 +16,39 @@ import (
 
 var infiniteResourceLimits = rcmgr.InfiniteLimits.ToPartialLimitConfig().System
 
+func makeResourceMgrs(maxMemory uint64, maxFD int, connMgrHighWater int, separateDHT bool) (bitswapHost, dhtHost network.ResourceManager, err error) {
```

Comment:
The changes here might be overkill and is pretty guesstimated, just wanted something other than disabling the resource manager for people to test with.

Recommendations appreciated 🙏